app/src/main/res/values/strings.xml

```diff
@@ -145,6 +145,7 @@
     <string name="snackbar_imagecropped">Images cropped</string>
     <string name="cropImage_activityTitle">Crop Image</string>
     <string name="snackbar_pdfCreated">PDF created!</string>
+    <string name="snack_bar_empty_txt_in_pdf">The chosen PDF has no text to extract!</string>
```

Comment:
can you add this string to other locale files also (in english only)? - just an addition

Reply:
Yes, I've added localisation for the new string